packages/orbit-tailwind-preset/src/presets/components/index.ts

```diff
@@ -206,7 +206,11 @@ const cfg = (options?: Options): Config => {
           addVariant("not-first", "&:not(:first-child)"),
           addVariant("type-even", "&:nth-of-type(even)"),
           addVariant("type-odd", "&:nth-of-type(odd)"),
-          addVariant("target-blank", "&[target='_blank']")
+          addVariant("target-blank", "&[target='_blank']"),
+          addVariant(
+            "safari",
```

Comment:
We add this but we don't do anything with it… Then, why? 💭

Reply:
I planned to add that after initial review of adding gap